### PR TITLE
Update adding or removing extensions procedure

### PR DESCRIPTION
**Pull request by svor**
## What does this pull request change?
Update adding or removing VS Code extensions procedure

Related MR: https://gitlab.cee.redhat.com/red-hat-developers-documentation/devspaces-documentation/-/merge_requests/83

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/CRW-6542

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/partials/proc_adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance.adoc [hunk 1]
```diff
@@ -35,14 +43,15 @@ If the extension publisher is unavailable or unwilling to publish the extension
...
-** To remove extensions, remove the publisher and extension names from the `openvsx-sync.json` file.
+** To add extensions, add the publisher, name and extension version to the `openvsx-sync.json` file.
+** To remove extensions, remove the publisher, name and extension version from the `openvsx-sync.json` file.
 ** Use the following JSON syntax:
 +
 [source,json,subs="+quotes"]
 ----
     {
-        "id": "__<publisher>__.__<extension>__"
+        "id": "__<publisher>__.__<extension>__",
```
Comment (tolusha) [inline modules/administration-guide/partials/proc_adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance.adoc]:
```suggestion
        "id": "__<publisher>__.__<name>__",
```

**Review by ibuziuk — APPROVED** [review]

**Comment by ibuziuk** [comment]
@deerskindoll please, back port for 3.15 once merged

**Comment by svor** [comment]
Converted to draft, I'm going to add more information

**Comment by svor** [comment]
@deerskindoll 
PR is ready for review

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**